### ISSUE TITLE
Ethereum block parser

### ISSUE THREAD

**Issue by tintinweb** [gcarq/rusty-blockparser#8]
Hi,

are there any plans on adding support for parsing raw ethereum blockdata?

-tin

**Comment by gcarq** [comment]
Yes, I'm planning to add ethereum, but this has no priority so I can't give an ETA. Is there any particular data structue you are interested in?

**Comment by tintinweb** [comment]
Not a problem. Would love to see eth support as I am particularly interested in parsing transaction data.

**Comment by tintinweb** [comment]
any updates?

//PS: lg aus österreich ;)

**Comment by craigathon** [comment]
Anything I can do to help with this? I assume it is more complicated than simply adding values to types.rs?

I also noticed the commented out Dash coin. Could you help me understand what makes that one different (x11)?

Thanks for the awesome project @gcarq

**Comment by wahid028** [comment]
Any update or someone can tell me which block parser would be the best for Ethereum blockchain
